src/types/credentials.ts

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * Matrix account credentials for a known user.
+ */
```

Comment:
Account credentials in my head would be user+password
This is more akin to Session/Device Auth data

Reply:
To challenge the assumption: we already call them credentials in `Lifecycle`. Granted, we say they're for a matrix client which might help a bit, but they are called credentials.

We don't have to keep this trend though.